tests/test_schema.py

```diff
@@ -4817,3 +4817,61 @@ def test_describe_escape(self):
             r"function test::foo() ->  std::str "
             r"using (SELECT r'\1');"
         )
+
+    def test_describe_poly_01(self):
+        self._assert_describe(
+            """
+            scalar type all extending str;
+
+            type Object {
+                property real -> bool;
+            }
+
+            function all() -> bool {
+                using (
+                    SELECT true
+                );
+            }
+            """,
+
+            'DESCRIBE OBJECT all AS TEXT',
+
+            """
+            function test::all() -> std::bool using (SELECT
+                true
+            );
+            scalar type test::all extending std::str;
+
+            # The following builtins are masked by the above:
+
+            # function std::all(vals: SET OF std::bool) ->  std::bool {
+            #     volatility := 'IMMUTABLE';
+            #     using sql
+            # ;};
```

Comment:
I don't think this is needed. IMO `# The std::all builtin is masked by the above:` would suffice. Interested users can always describe the fully qualified name and that description would be properly syntax highlighted.

Reply:
I was thinking about it. My reasoning is that masking isn't common and it's nicer if you get full info in a single command invocation rather than having to then look up separately. Basically this won't affect 99% schemas and will make the rare schemas that mask things a little easier to navigate and reason about.

Reply:
I'm fine to keep it the way you implemented. It's a bit too noisy for my taste, but I hear your reasoning too.

Reply:
@elprans thoughts?